src/packagedcode/pyrpm/rpm.py

```diff
@@ -3,17 +3,17 @@
 # vim:ts=4:sw=4:et
 
 # Copyright (c) M�rio Morgado
-# 
```

Comment:
It is usually best to avoid mixing formatting changes with actual code changes. It make the patches easier to review. ;)